database/readme.md

```diff
@@ -0,0 +1,26 @@
+
+# Testing Databases 
+
+## Stages 
+
+Include `stages.yaml` in your CI file for the required stages of a database service. 
+
+## Build Testing Postgres Databases 
+
+### Npm 
+
+Extending your build with from `.buildDatabaseWithNpm` will use npm with flyway to set up a docker image containing a postgres database that your tests can run against. 
```

Comment:
This wording is a bit misleading.  npm and flyway aren't as related as this makes it sound.  It will create your database schema with flyway and then run tests with npm test.